### PR TITLE
a little somethnig

### PR DESCRIPTION
**Pull request by joestringer**
- Add cool stuff
- woops

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Litee-gh-testing
 
 this is a test repository for developing `litee-gh.nvim`
+
+with some totally cool new stuff in it
```
Comment (joestringer) [inline README.md:7]: tbh I'm not sure quite how cool this is.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
 # Litee-gh-testing
 
 this is a test repository for developing `litee-gh.nvim`
+
+woops
```
Comment (joestringer) [inline README.md:5]: ok this time I did `GHStartReview` first

Comment (joestringer) [reply README.md:5]: This was posted about the cool stuff, not the woopsies

Comment (ldelossa) [reply README.md:5]: So in gh.nvim you created this comment on line 7?

**Comment by joestringer** [comment]
Well actually it was line 5 when I was commenting, because it's line 5 on
the first commit. Then the second commit moved the content to line 7.

**Comment by ldelossa** [comment]
@joestringer so in gh.nvim, you have the first commit opened right? You make a comment on line 5, but then it puts it on line 7.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
 # Litee-gh-testing
 
 this is a test repository for developing `litee-gh.nvim`
+
```
Comment (ldelossa) [inline README.md:4]: A new thread

**Review by ldelossa — APPROVED** [review]
bleh

File: README.md (hunk 1 above)
Comment (ldelossa) [inline README.md:7]: test

**Review by ldelossa — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
...
```
Comment (ldelossa) [inline README.md:4]: multi line

File: README.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
...
+woops
+
+with some totally cool new stuff in it
```
Comment (ldelossa) [inline README.md:7]: Yup

File: README.md (hunk 1 above)
Comment (ldelossa) [inline README.md:7]: yup yup

Comment (joestringer) [inline README.md:7]: Creating a thread about that totally cool stuff

Comment (joestringer) [reply README.md:7]: Looks good from gh.nvim

Comment (joestringer) [reply README.md:7]: Looks good from github :+1:

File: README.md (hunk 2 above)
Comment (joestringer) [inline README.md:5]: I think this is multiline?

File: README.md [hunk 6]
```diff
@@ -2,4 +2,6 @@
 
 this is a test repository for developing `litee-gh.nvim`
```
Comment (ldelossa) [inline README.md]: comment created @ HEAD~1

**Comment by joestringer** [comment]
Yep!

**Pull request closed without merging**